src/lib/utils/paths.ts

```diff
@@ -20,6 +19,14 @@ export function getPublicPath(assetPath: string): string {
  * @params path
  * @returns string
  */
-export function getDefaultHostPath(path: string): string {
-    return `${DEFAULT_HOST_URL}${path}`
+export function getWordpressUrl(path: string): string {
+
+    // the domain is just the current window location
+    const domain: string = `${window.location.protocol}//${window.location.host}`
```

Comment:
@testflyjets actually, this part is causing issues.

Eg. for platform-ui, when you click "talent" in the main nav, it will redirect you to <current location host>/talent: https://platform-ui.topcoder-dev.com/talent. Because it takes the current url, and that's not only for WP pages, it's for anything that integrates the uninav. 

I'll find a solution for it.

Reply:
Okay, thanks for catching that.